app/views/hq/orphans/_form.html.erb

```diff
@@ -116,7 +116,7 @@
       </div>
       <div class="form-group">
         <%= f.label_for_field :guardian_id_num %>
-        <%= f.text_field :guardian_id_num, class: "form-control" %>
+        <%= f.number_field :guardian_id_num, class: "form-control" %>
```

Comment:
The `number_field` is more useful when it is used for fields that represent some kind of quantity.
Also changes to the behavior of the application should be discussed with the client and dev team first.
Also this is not related to the current task.
I would prefer to keep this as it was.